### DISCUSSION TITLE
MRU Showing on One Computer, Not the Other

### DISCUSSION THREAD

**Discussion by skeezix47** [Open-Shell/Open-Shell-Menu#1193] [Q&A] [answered]
Using Classic style on two computers, one a Win10 desktop and the other a Win11 laptop. The most recently used items show next to the menu options on the laptop, but not on the desktop (the same was true for the laptop's Win10 predecessor). I backed up the settings for both systems and compared the .xml files, and they're identical.

Any insights? Thanks.

**Comment by ge0rdi** [comment, marked as answer]
Have a look at #149.
You may have app launch tracking disabled in Windows settings on the other machine.
